db/migration/src/lib.rs

```diff
@@ -4,6 +4,7 @@ mod m20251124_00001_tickets_create_ticket_stats;
 mod m20251124_00002_tickets_create_outgoing_ticket_index;
 mod m20251124_00003_tickets_create_ticket;
 mod m20251124_00004_peers_create_peers;
+mod m20251212_00005_peers_remove_everything;
```

Comment:
I would still rather prefer to remove the original migration, since we have not officially released 4.0 yet. 

Also the intitial peers migration is effectively a dead code now since nothing brings it up anymore (you removed the peers db).